tox.ini

```diff
@@ -6,13 +6,14 @@ install_requires =
 name = sphinxcontrib
 
 [tox]
-envlist = lint,py{34,27}
+envlist = lint,py{35,27}
```

Comment:
If only one Python 3 version is used by default, I would rather make it the latest. Otherwise, why not `py{37,36,35,27}`? You can use `skip_missing_interpreters = True`.

Reply:
I have never used `tox` before so I am pretty clueless on the "correct" way to do this. I simply followed the existing pattern. If you would like it changed to something you prefer simply tell me what that is. I will gladly make this the latest version of python 3 instead of the oldest or anything else if there is a reason to do so. As a side note, I think changing this simply due to preference is unremunerative.

Reply:
I think the most correct way to do it is to add all supported Python versions. `skip_missing_interpreters` could be friendlier for developers, but definitely not required; it will simply not run tests against missing interpreters in your system/environment.

Reply:
Updated tox.ini